samples/menus/nav-drawer/add-positions-navbar/index.html

```diff
@@ -0,0 +1,49 @@
+<html>
+  <head>
+    <title>NavDrawer Add Positions and Navbar</title>
+    <meta charset="UTF-8" />
+
+    <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png" >
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v5.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="root" style="display: flex;">
+      <igc-nav-drawer id="navDrawer" style="position: relative">
+          <igc-nav-drawer-header-item>
+              Sample Drawer
+          </igc-nav-drawer-header-item>
+
+          <igc-nav-drawer-item>
+              <igc-icon slot="icon" name="home"></igc-icon>
+              <span slot="content">Home</span>
+          </igc-nav-drawer-item>
+
+          <igc-nav-drawer-item>
+              <igc-icon slot="icon" name="search"></igc-icon>
+              <span slot="content">Search</span>
+          </igc-nav-drawer-item>
+      </igc-nav-drawer>
+
+      <div id="content" style="width: 100%">
+        <igc-radio-group id="radio-group" alignment="horizontal">
+          <igc-radio name="position" value="start" label-position="after" checked="true">Start</igc-radio>
```

Comment:
The correct way for setting boolean attributes.
```suggestion
          <igc-radio name="position" value="start" label-position="after" checked>Start</igc-radio>
```